### PR TITLE
{lib}[GCCcore/11.3.0] jxrlib v1.1, imagecodecs v2022.9.26, LERC v4.0.0, LibLZF v3.6, libavif v0.11.1, zfp v1.0.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by deniskristak**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/imagecodecs/imagecodecs-2022.9.26-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+# update: Thomas Hoffmann (EMBL), Denis Kristak (Inuits)
+easyblock = 'PythonBundle'
+
+name = 'imagecodecs'
+version = '2022.9.26'  # opting for older version because of compatibility issues with LibTIFF
```
Comment (boegel) [inline easybuild/easyconfigs/i/imagecodecs/imagecodecs-2022.9.26-foss-2022a.eb:7]: @deniskristak What kind of problems did you run into exactly here with the latest `imagecodecs` version?

Comment (deniskristak) [reply easybuild/easyconfigs/i/imagecodecs/imagecodecs-2022.9.26-foss-2022a.eb:7]: @boegel I think it was these errors: https://github.com/vscentrum/vsc-software-stack/blob/wip/76_cellpose/old_logz/easybuild-run_cmd-82n3t0f4.log
( `error: unknown type name TIFFOpenOptions` )
I didn't find much on how to solve the issue as such, but changing simply reverting to an older version seems to have worked.

File: easybuild/easyconfigs/i/imagecodecs/imagecodecs-2022.9.26-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    ('HDF5', '1.12.2'),
+    ('h5py', '3.7.0'),
+    ('libheif', '1.12.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('bitshuffle', '0.3.5', {
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/i/imagecodecs/imagecodecs-2022.9.26-foss-2022a.eb]: fixed in [fc2c1ad](https://github.com/easybuilders/easybuild-easyconfigs/pull/18003/commits/fc2c1ad011c18a304fdb28d110ee510c4f69083e)

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3159.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/93007cb0781f5706a10acf35f3006b7b for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3159.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/f925b70ae2642829c46a641a474650ac for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 16 out of 16 (6 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/50f6ff3032d9c03a6e657acf1273ac0d for a full test report.

**Review by casparvl — APPROVED** [review]
Lgtm!

**Comment by casparvl** [comment]
Going in, thanks @deniskristak!

**Pull request merged**